src/javascripts/modules/demos.js

```diff
@@ -0,0 +1,37 @@
+export default class Demos {
+  constructor(el) {
+    this.el = el
+    this.setVars()
+    this.createObserver()
+  }
+
+  setVars() {
+    this.hasAppeared = false
+    this.demoOne = document.querySelector('.demo-one .messages')
+    this.demoTwo = document.querySelector('.demo-two .messages')
+    this.messagesLeft = document.querySelectorAll('.message-left')
+  }
+
+  createObserver() {
+
+    this.options = {
+      root: null,
+      rootMargin: "0px",
+      threshold: 0.5
+    }
```

Comment:
does this need to be set as a class property or can it just be local to this method (ie. is it used somewhere else)?

Reply:
Is this specifically about the options object? In that case you're right, it can just be local, I didn't catch that